palantirnet/drupal-skeleton
# test drupal 10

### Description

_A description of the changes. Consider including images/gifs to demonstrate behaviors (e.g. using [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture))._

### Testing instructions

1. `composer create-project palantirnet/drupal-skeleton drupal10 dev-drupal-10 --no-interaction`
2. `cd drupal10`
3. Edit `.ddev/config.yaml` and change the `name` to `drupal10`
4. `ddev start`
5. `ddev ssh`
6. `vendor/bin/the-build-installer`
7. Visit https://drupal10.ddev.site/ and verify the site is running.
8. Log in, and verify the site is running Drupal 10.

### Remaining tasks

- [x] Review and merge https://github.com/palantirnet/the-build/pull/201
- [x] Create a new release of `the-build`
- [x] Change line 30 here: https://github.com/palantirnet/drupal-skeleton/pull/126/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R30 to use `^4@beta` (or `^4` if there is a stable release).
